serializations.md

```diff
@@ -26,17 +26,18 @@ not divided into a header and a list of transactions;
 All field sizes are statically known and can be constructed directly as
 a byte array.
 
-| Fieldname | Size (bytes) |
-| --- | --- |
-| version    | 8    |
-| height     | 8    |
-| prev_hash  | 32   |
-| state_hash | 32   |
-| miner      | 32   |
-| target     | 8    |
-| pow        | 168  |
-| nonce      | 8    |
-| time       | 8    |
+| Fieldname  | Size (bytes) |
+| --- | ---  |
+| version     | 8    |
+| height      | 8    |
+| prev_hash   | 32   |
+| state_hash  | 32   |
+| beneficiary | 32   |
```

Comment:
Where shall beneficiary be serialized? Before or after miner? This seems the opposite of https://github.com/aeternity/epoch/pull/1323/files#diff-3422fb1f1624d1021226c01eed4d0cd0R155

Reply:
@lucafavatella Yes, good catch.